dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/channel/internal/HttpChannelConfig.java

```diff
@@ -2984,6 +3012,13 @@ public boolean onlySameSiteStar() {
         return this.onlySameSiteStar;
     }
 
+    /*
+     * Returns a boolean which indicates whether Partitioned should be added the SameSite=None cookies. 
```

Comment:
`should be added to the SameSite=None cookies.`

Reply:
Will fix in the next PR